add new user controllers

tarik310 opened the pull request:
add new user controllers close #71 

• add getMyInfo controller shows user profile.
• add getCookPage to get cooks' address, location, and dishes.
• add deleteMyaccount to delete all documents related to the user.

tarik310 committed:
add new user controllers

tarik310 committed:
fix getcookpage controller

OmarQaqish reviewed (commented):
Great work, @tarik310. Just a couple of notes: 1. since I removed the isAdmin field from the user model, you don't need to set it to 0 in the queries anymore, and 2. this is not important but just for consistency, let's please stick to camel case when naming, like DeleteMyAccount, UserInfo, etc.

tarik310 committed:
fix deleteMyAccount controller

tarik310 commented:
> Great work, @tarik310. Just a couple of notes: 1. since I removed the isAdmin field from the user model, you don't need to set it to 0 in the queries anymore, and 2. this is not important but just for consistency, let's please stick to camel case when naming, like DeleteMyAccount, UserInfo, etc.

i fixed the camel case and added deleting orders and carts statements
